using-kubecost/navigating-the-kubecost-ui/savings/savings-actions.md

```diff
@@ -155,6 +155,70 @@ In the second collapsible tab, you can configure continuous cluster sizing.
 
 Select _Enable Auto-Resizing Now_ to finalize.
 
+## Creating an action via *values.yaml*
+
+You can also optionally configure Actions via your Kubecost [*values.yaml*](/install-and-configure/install/helm-install-params.md) file. Configure the following section as needed for any actions you wish to set up. For more information of any action-specific field, see the individual UI configuration sections above for more information.
+
+```
+actionConfigs:
+    # this configures the Kubecost Cluster Turndown action
+    # for more details, see documentation at https://github.com/kubecost/cluster-turndown/tree/develop?tab=readme-ov-file#setting-a-turndown-schedule
+    clusterTurndown: []
+      # - name: my-schedule
```

Comment:
brett, I would uncomment these if your goal is to display enabled config. They are only commented since we deploy them in the active values file and obviously did not want them to be enabled

Reply:
I would rather remove those sections entirely than uncomment them, I feel they make this section a bit bloated

Reply:
that also works

Reply:
Ok, I've made several updates